build/httpd/restrict-wp-admin.conf

```diff
@@ -1,12 +1,15 @@
 # This will restrict the access of wp-admin to the EPFL campus
 # and private addresses (for local testing)
+# and docker addresses (for functional testins)
 <LocationMatch ".*/wp-login.php.*">
     Require ip 172.16.0.0/12
     Require ip 128.178.0.0/16
     Require ip 128.179.0.0/16
+    Require ip 172.17.0.0/16
```

Comment:
172.16.0.0/12 (first line) includes:
172.16.0.1 to
172.31.255.254
http://jodies.de/ipcalc?host=172.16.0.0&mask1=12
So the line Require ip 172.17.0.0/16 should not be required